File: semester I/Praktikum Pemrograman I/Modul1_JovanGilbertNatamasindah_2310817310002/C/PRAK104-2310817310002-JovanGilbertNatamasindah.c
#include <stdio.h>

void main () {
    int SEPATUA = 400000;
    int SEPATUB = 350000;

    int DISKONA = 400000 * 13/100;
    int DISKONB = 350000 * 21/100;
    
    int HASILA = SEPATUA - DISKONA ;
    int HASILB = SEPATUB - DISKONB ;

    printf("Harga Sepatu A adalah %d\n", SEPATUA);
    printf("Harga Sepatu B adalah %d\n", SEPATUB);
    printf("Sepatu A mendapat diskon 13%% sehingga harganya menjadi %d\n", HASILA );
    printf("Sepatu B mendapat diskon 21%% sehingga harganya menjadi %d\n", HASILB );
}

File: semester I/Praktikum Pemrograman I/Modul1_JovanGilbertNatamasindah_2310817310002/Python/PRAK104-2310817310002-JovanGilbertNatamasindah.py
SEPATUA = 400000
SEPATUB = 350000

DISKONA = 400000 * 13/100
DISKONB = 350000 * 21/100

HASILA = round(SEPATUA - DISKONA) 
HASILB = round(SEPATUB - DISKONB)
    
print("Harga Sepatu A adalah ", SEPATUA)
print("Harga Sepatu B adalah ", SEPATUB)
print("Sepatu A mendapat diskon 13% sehingga harganya menjadi ", HASILA)
print("Sepatu B mendapat diskon 21% sehingga harganya menjadi ", HASILB)